deployer/infra_components/cluster.py

```diff
@@ -94,6 +94,35 @@ def deploy_support(self, cert_manager_version, debug):
             )
             print_colour("Done!")
 
+            # Patch the tigera operator to remove the NoSchedule toleration
+            # otherwise it will schedule on tainted nodes
+            print_colour("Patching tigera operator...")
+            patch_tolerations = {
+                "spec": {
+                    "template": {
+                        "spec": {
+                            "tolerations": [
+                                {"effect": "NoExecute", "operator": "Exists"},
```

Comment:
This is coming from https://github.com/projectcalico/calico/blob/da6b089ff18f2e4ab104def0f78e68c8741fc221/manifests/tigera-operator.yaml#L535-L539

Wasn't sure if I should keep it, but for now it doens't bother us _I think_